google-summer-of-code/gsoc-2022.md

```diff
@@ -80,7 +80,7 @@ Contributors are welcome to participate in any talks and discussions in the Knat
 
 * Knative users group: https://groups.google.com/g/knative-users
     * Preferred communication channel. Introduce yourself here as a candidate contributor.
-* Slack: https://slack.knative.dev/ Explore the channels yourselves, however, make sure you're checking these:
+* Slack: https://slack.cncf.io/ Explore the channels yourselves, however, make sure you're checking these:
     * `#serving`: Knative Serving related discussions
     * `#eventing`: Knative Eventing related discussions
```

Comment:
We should update the channel names here too (and probably should add `#functions` to the list)